new_model_unseen_template/README.md

```diff
@@ -0,0 +1,47 @@
+Here we train a new VAE model. Then simulate a background dataset using a template experiment that is **not included** in the training dataset (i.e. the datasets used to train the VAE model).
+
+## Usage
+1. Create a configuration and metadata files for your analysis. 
+
+See [example configuration file]().
+When editing this configuration file, use a text editor instead of excel.
+
+The definition for each paramter in the configuation file is describe below:
+
+Parameters for input data files and directories to create
+
+| Name | Description |
+| :--- | :---------- |
+| base_dir| str: Root directory containing analysis subdirectories. By default the path is one level up from where the scripts are run.|
+| local_dir| str: Parent directory on local machine to store intermediate results. Make sure to end path name with "/"|
+| dataset_name| str: Name for analysis directory, which contains the notebooks being run. For example, this analysis directory is named "new_model_unseen_template".|
+| raw_template_filename | str: Downloaded template gene expression data file. The input dataset should be a matrix that is sample x gene. The file should tab-delimited. The gene ids should be HGNC symbols (if using human data) or PA numbers (if using *P. aeruginosa* data). The input dataset should be generated using the same platform as the model you plan to use (i.e. RNA-seq or array). The expression values are expected to have been uniformly processed and can be estimated counts (RNA-seq) or log2 expression (array).|
+| raw_compendium_filename | str: Downloaded compendium gene expression data file. The input dataset should be a matrix that is sample x gene. The file should tab-delimited. The gene ids should be HGNC symbols (if using human data) or PA numbers (if using *P. aeruginosa* data). The input dataset should be generated using the same platform as the model you plan to use (i.e. RNA-seq or array). The expression values are expected to have been uniformly processed and can be estimated counts (RNA-seq) or log2 expression (array).|
```

Comment:
This isn't necessarily true, right? Sophie should work on any compendium, whether it's PA or Human or any organism? The only specification should be that the genes are the same in the compendium file as they are in the gene expression template experiment file?

Reply:
Yes you're correct. I'll remove this text